gen_cfg_section.py

```diff
@@ -0,0 +1,122 @@
+import argparse
+import os
+import re
+from urllib.request import urlopen
+
+import yaml
+
+
+def parse_cli_args():
+    parser = argparse.ArgumentParser(
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter
+    )
+    parser.add_argument(
+        'api_versions',
```

Comment:
Not needed. You can detect the version directly from the docker socket. Then you can use that single version instead of worrying about many.

Reply:
To be clear: you would like a version of this script to run in the proxy container, interrogate the current Docker API version, load the appropriate (pre-downloaded) Swagger YAML file, generate the haproxy.cfg file, and start the proxy?

Reply:
Yes! :blush: